incubator/virtualcluster/cmd/kubectl-vc/root.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/urfave/cli"
```

Comment:
Any reason we went with `urfave`? Isn’t the rest of vc using cobra?

Reply:
i think urfave is much easier to construct a CLI with multi subcommands 😂
kubectl vc is a very simple tool with clean implementation

Reply:
Gotcha, as we dev this we might want to strive for some consistency across the components so that devs don't need to relearn and we can reuse the same packages.

Reply:
I'm referring to things like having a shared flags implementation cause all of the components use different flags eg `master-kubeconfig` `kubeconfig`, while not in this cli yet I'm sure we're going to hit this eventually.